doc/manual/team.rst

```diff
@@ -375,18 +375,22 @@ for different programming languages.
    :language: c
    :caption: *A solution in C*
 
+.. raw:: latex
+
+   \clearpage
+
 .. literalinclude:: ../examples/example.cc
    :language: cpp
    :caption: *A solution in C++*
 
-.. raw:: pdf
-
-   PageBreak
-
 .. literalinclude:: ../examples/example.java
    :language: java
    :caption: *A solution in Java*
 
+.. literalinclude:: ../examples/example.kt
+   :language: java
```

Comment:
This should be `kotlin` as the highlighting fails with the Java lexer.

Reply:
Oops, copy&paste error. Fixed.